recipes/watchdog-wms/run_test.sh

```diff
@@ -7,7 +7,7 @@ VERSION="${PKG_NAME}-${PKG_VERSION}-${PKG_BUILDNUM}"
 OUT="${PREFIX}/share/${VERSION}"
 PATH="$PATH:$OUT/core_lib/wrapper/"
 
-function timeoutbash() { bash -c '(sleep 10; kill -9 $$ 2>/dev/null) & exec $@' $0 $@; }
+function timeoutbash() { bash -c '(sleep 180; kill -9 $$ 2>/dev/null) & exec $@' $0 $@; }
 
 # basic test
 timeoutbash watchdog-cmd --help 2>&1>/dev/null
```

Comment:
Does the program not terminate itself after it output the help information?

Reply:
Sure, it does. I included the timeout command for the more advanced test.
("not terminating" could be caused by a bug in the github version in that case.)